### PR TITLE
Created README.md file

### PR DESCRIPTION
**Pull request by weskeyz**

### CODE REVIEW

File: _data/weskeyz/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Weskeyz's Zarr collection
```
Comment (sanketverma1704) [inline _data/weskeyz/README.md]: Please hyperlink your GitHub username here.

File: _data/weskeyz/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Tweet mentions:
```
Comment (sanketverma1704) [inline _data/weskeyz/README.md:3]: You can use the markdown sub-heading format here. (##)

File: _data/weskeyz/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+https://twitter.com/pleiszenburg/status/1197567468845309952?s=20&t=pVprlZ-mbMk0cOkxOplfLw
+https://twitter.com/miguelmahechag1/status/1561038886062751746?s=20&t=pVprlZ-mbMk0cOkxOplfLw
+
+Zarr Data visualization
+![Opera Snapshot_2022-10-21_101658_twitter com](https://user-images.githubusercontent.com/94924532/197161914-c01c3048-c015-4692-a1c5-a6c1c5f3064f.png)
+
+Code snippets
+Zarr API For Accessing HRRR Data (University of Utah) - https://mesowest.utah.edu/html/hrrr/zarr_documentation/html/zarr_api_multiple_hrrr_variables.html
+
+%%time
```
Comment (sanketverma1704) [inline _data/weskeyz/README.md]: Please put the code snippet in code brackets using (```<CODE>```).

File: _data/weskeyz/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+
+ax = plt.axes(projection=ccrs.PlateCarree())
+ax.contourf(chunk_index.longitude, chunk_index.latitude, thousand_mb_tmp)
+ax.coastlines()
+
+plt.show()
+plt.close()
+
```
Comment (sanketverma1704) [inline _data/weskeyz/README.md:47]: Also, can you please add the code output at the bottom? Thanks.

**Comment by weskeyz** [comment]
Okay
Understood

**Comment by MSanKeys963** [comment]
Hi @weskeyz. Thanks for sending the PR.
I've left some comments, please check them.

**Comment by weskeyz** [comment]
Okay, thank u

**Comment by weskeyz** [comment]
Hi Sanket

You asked me to put my code snippet in core brackets..

I'm a little confused there, do I put each like of the code in code
brackets?

**Comment by MSanKeys963** [comment]
Hi @weskeyz.
Please refer to the documentation [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

**Comment by MSanKeys963** [comment]
Hi @weskeyz, do you plan to work on this?

**Comment by MSanKeys963** [comment]
Closing this for now. Let me know if you plan to work on this @weskeyz.
Thanks!

**Pull request closed without merging**